src/static/css/nav-secondary.less

```diff
@@ -128,15 +148,32 @@
 */
 
 .nav-secondary {
-    margin-bottom: unit(30px / @base-font-size-px, em);
+    margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
+
+    .respond-to-max(800px, {
+        border-bottom: 1px solid @gray-20;
+    });
+
+    // Inset horizontal rule between "IN THIS SECTION" text and page links.
+    & .inset-divider {
+      margin: 0 @grid_gutter-width / 2 @grid_gutter-width / 2;
+      border: 0;
+      height: 1px;
+      background-color: @gray-20;
+
+      // Only shown on tablet/mobile sizes.
+      .respond-to-min(801px, {
+        display: none;
+      });
+    }
 
     &_list {
         margin: 0;
         padding: 0;
     }
-
-    &_item {
+    &_item, &_item__child {
         list-style: none;
+        .webfont-regular();
```

Comment:
Could we move both of these to the `_list` above so it just applies to the entire list, no matter if there are children or not?